test-suite/js/src/test/scala/org/scalajs/testsuite/library/WrappedMapTest.scala

```diff
@@ -0,0 +1,90 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.testsuite.library
+
+import org.junit.Assert._
+import org.junit.Assume.assumeTrue
+import org.junit.{BeforeClass, Test}
+
+import scala.collection.mutable
+import scala.scalajs.{LinkingInfo, js}
+
+object WrappedMapTest {
+  @BeforeClass
+  def assumeRuntimeSupportsSet(): Unit = {
+    assumeTrue("Assume ES6", LinkingInfo.assumingES6)
```

Comment:
I do not know why, but `ReferenceError: Symbol is not defined` happens unless symbol availability is assumed here.

Reply:
The failing tests are `MapTest` and `SetTest`, not this one. This one works with the assumption on `assumeES6`. You should also use the same assumption in `WrappedMapTest` and `WrappedSetTest`, where you haven't changed it to `assumeES6` yet.